decidim-debates/lib/decidim/debates/component.rb

```diff
@@ -21,7 +21,7 @@
     settings.attribute :scopes_enabled, type: :boolean, default: false
     settings.attribute :scope_id, type: :scope
     settings.attribute :comments_enabled, type: :boolean, default: true
-    settings.attribute :comments_max_length, type: :integer, required: false
+    settings.attribute :comments_max_length, type: :integer, default: 1000, required: true
```

Comment:
Required `true` is fine to make it consistent with others but I would leave the `default` out because otherwise we are forcing the value on components (rather than using the default).
```suggestion
    settings.attribute :comments_max_length, type: :integer, required: true
```